nulib/meadow
# Add Reading Room metadata to UI

# Summary 
This adds support for Reading Room boolean metadata values in the UI.

# Specific Changes in this PR
- On individual Work page users can update Reading Room value
- On Batch Edit users can update Reading Room value
- Reading Room facet added

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test
1. Go to a Work page, and on the Administrative tab, find "Reading Room".  Update the value and save.
2. Go to the search page and facet on "Reading Room".  Notice if your Work updated correctly.
3. Select multiple items from the Search page for Batch Edit.  Update Reading Room values for selected items.
4. Go back to search page, facet on "Reading Room" and notice if your updates are reflected.

Also please let developers know if there are any special instructions to test this in the development environment. 

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
